src/static/js/media.js

```diff
@@ -784,6 +790,23 @@ $(function() {
                 buttons: signbuttons
             });
 
+            $("#dialog-signlink").dialog({
+                autoOpen: false,
+                width: 550,
+                modal: true,
+                dialogClass: "dialogshadow",
+                show: dlgfx.edit_show,
+                hide: dlgfx.edit_hide
+            });
```

Comment:
future reference - you don't need to do this when you're using show_okcancel_dialog